acceptance/apps/rails_test.go

```diff
@@ -89,49 +89,46 @@ var _ = Describe("RubyOnRails", func() {
 		out, err = env.Epinio("", "apps", "env", "set", rails.Name, "RAILS_MASTER_KEY", rails.MasterKey)
 		Expect(err).ToNot(HaveOccurred(), out)
 
-		// Create a database for Rails
-		out, err = proc.Run("", false, "helm", "repo", "add", "bitnami", "https://charts.bitnami.com/bitnami")
+		// Create a custom service catalog
+		out, err = proc.Kubectl("apply", "-f", testenv.TestAssetPath("my-postgresql-custom-svc.yaml"))
 		Expect(err).ToNot(HaveOccurred(), out)
 
-		// Update helm repos
-		out, err = proc.Run("", false, "helm", "repo", "update")
+		serviceName = names.TruncateMD5(catalog.NewCatalogServiceName(), 40)
+
+		// Create a database for Rails
+		out, err = env.Epinio("", "service", "create", "my-postgresql", serviceName)
```

Comment:
always use random names for things in tests. We have helpers that generate names in other tests. I think you need `catalog.NewServiceName()` here.

Reply:
Don't know if I have to wait on #1370 to merge or if it can be done after...

Reply:
> always use random names for things in tests. We have helpers that generate names in other tests. I think you need `catalog.NewServiceName()` here.

This commit https://github.com/epinio/epinio/pull/1405/commits/8f3506bd13c50d1741244277ff9387881c235c66 addresses your comment @jimmykarily